src/ontology/mondo-edit.obo

```diff
@@ -348133,20 +348135,20 @@ intersection_of: has_characteristic MONDO:0024495 ! tumor grade 1 or 2, general
 
 [Term]
 id: MONDO:0015068
-name: neuroendocrine tumor of rectum, well differentiated, low or intermediate grade
```

Comment:
This name should be added as an exact synonym with MONDO:0015068 as an x-ref (and probably NCIT x-ref since this is where it came from)

Reply:
Comments addressed.